Source/LibationWinForms/Form1.cs

```diff
@@ -262,8 +262,8 @@ private void configAndInitAutoScan()
 					.Where(a => a.LibraryScan)
 					.ToArray();
 
-					// in autoScan, new books SHALL NOT show dialog
-					await LibraryCommands.ImportAccountAsync(Login.WinformLoginChoiceEager.ApiExtendedFunc, accounts);
+				// in autoScan, new books SHALL NOT show dialog
+				await Invoke(async () => await LibraryCommands.ImportAccountAsync(Login.WinformLoginChoiceEager.ApiExtendedFunc, accounts));
```

Comment:
This was throwing a cross-thread error after it's first 5 minutes elapsed.

Reply:
Damn. It didn't do that for me. Thanks for the fix